recipes/tesseract/all/conanfile.py

```diff
@@ -0,0 +1,111 @@
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
+        if self.options.with_training:
+            # do not enforce failure and allow user to build with system cairo, pango, fontconfig
+            self.output.warn("*** Build with training is not yet supported, continue on your own")
+
+    def configure(self):
+        # Exclude old compilers not supported by tesseract
+        compiler_version = Version(str(self.settings.compiler.version))
+        if (self.settings.compiler == "gcc" and compiler_version < "5") or \
+                (self.settings.compiler == "clang" and compiler_version < "5"):
+          raise ConanInvalidConfiguration("Compiler is not supported")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        cmake = self._cmake = CMake(self)
+        cmake.definitions['BUILD_TRAINING_TOOLS'] = self.options.with_training
+        cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
+        cmake.definitions["STATIC"] = not self.options.shared
+        # Use CMake-based package build and dependency detection, not the pkg-config, cppan or SW
+        cmake.definitions['CPPAN_BUILD'] = False
+        cmake.definitions['SW_BUILD'] = False
+
+        # avoid accidentally picking up system libarchive
+        cmake.definitions['CMAKE_DISABLE_FIND_PACKAGE_LIBARCHIVE'] = True
+
+        # Set Leptonica_DIR to ensure that find_package will be called in original CMake file
+        cmake.definitions['Leptonica_DIR'] = self.deps_cpp_info['leptonica'].rootpath
+
+        cmake.configure(build_folder=self._build_subfolder)
+        return cmake
+
+    def build(self):
+        # Use generated cmake module files
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "CMakeLists.txt"),
+            "find_package(Leptonica ${MINIMUM_LEPTONICA_VERSION} REQUIRED CONFIG)",
+            "find_package(Leptonica ${MINIMUM_LEPTONICA_VERSION} REQUIRED)")
+        # Variable Leptonica_LIBRARIES does not know about its dependencies which are handled only
+        # by exported cmake/pc files which are not used by Conan.
+        # Therefore link with exported target from the autogenerated CMake file by the cmake_find_package
+        # that contains information about all dependencies
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "CMakeLists.txt"),
+            "${Leptonica_LIBRARIES}",
+            "Leptonica::Leptonica")
+
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses", ignore_case=True, keep_path=False)
```

Comment:
```suggestion
        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
```

Not required, there is only one license file and it's in source subfolder.